ru/browser/browser/preferences/zen-preferences.ftl

```diff
@@ -82,35 +101,42 @@ zen-pinned-tab-manager-reset-close-shortcut-option =
     .label = Reset URL
 zen-pinned-tab-manager-close-close-shortcut-option = 
     .label = Закрыть вкладку
-pane-zen-workspaces-header = Рабочие области
-zen-settings-workspaces-header = Общие настройки для рабочих областей
-zen-settings-workspaces-description = С рабочими областями вы можете использовать несколько сеансов просмотра одновременно!
+
+pane-zen-workspaces-header = Пространства
+zen-settings-workspaces-header = Общие настройки для пространств
+zen-settings-workspaces-description = С пространствами вы можете использовать несколько сеансов просмотра одновременно!
 zen-settings-workspaces-enabled = 
-    .label = Включить рабочие области (экспериментальные)
+    .label = Включить пространства (экспериментально)
 zen-settings-workspaces-hide-default-container-indicator = 
     .label = Скрыть индикатор контейнера по умолчанию на панели вкладок
 zen-settings-workspaces-allow-pinned-tabs-for-different-workspaces = 
-    .label = Разрешить рабочие области иметь собственные закрепленные вкладки
+    .label = Разрешить пространству иметь собственные закрепленные вкладки
+
 zen-key-unsafed = Ярлык не сохранен! Пожалуйста, сохраните его, нажав на клавишу "Escape" после повторного ввода.
 zen-key-conflict = Конфликт с другим ярлыком
+
 pane-zen-theme-title = Настройки темы
+
 zen-vertical-tabs-title = Расположение боковой панели и вкладок
 zen-vertical-tabs-header = Вертикальные вкладки
-zen-vertical-tabs-description = Управляйте своими вкладками в вертикальном макете
+zen-vertical-tabs-description = Управляйте своими вкладками в вертикальном расположении
 zen-vertical-tabs-show-expand-button = 
     .label = Показать кнопку расширения
 zen-vertical-tabs-newtab-on-tab-list = 
-    .label = Show New Tab Button on Tab List
+    .label = Показывать кнопку "Новая вкладка" в списке вкладок
 zen-vertical-tabs-newtab-top-button-border = 
-    .label = Show separation border for when the new tab button is at the bottom
+    .label = Показывать разделительную линию, когда она находится внизу
```

Comment:
We generally re-state the subject on every entry in the list, other than if it is stated in the entry-group header.

```suggestion
    .label = Показывать линию-разделитель, когда кнокпа "Новая вкладка" находится внизу
```